lib/Controller/RoomController.php

```diff
@@ -1076,6 +1076,14 @@ protected function formatParticipantList(array $participants, bool $includeStatu
 				$result['displayName'] = $participant->getAttendee()->getDisplayName();
 			} elseif ($participant->getAttendee()->getActorType() === Attendee::ACTOR_CIRCLES) {
 				$result['displayName'] = $participant->getAttendee()->getDisplayName();
+			} elseif ($participant->getAttendee()->getActorType() === Attendee::ACTOR_EMAILS) {
+				$result['displayName'] = $participant->getAttendee()->getDisplayName();
+				if ($this->participant->hasModeratorPermissions()) {
+					$result['status'] = IUserStatus::OFFLINE;
+					$result['statusIcon'] = null;
+					$result['statusMessage'] = $participant->getAttendee()->getInvitedCloudId();
```

Comment:
This is to bring the email to the "status line".
I would prefer to add a dedicated response data entry instead, but then clients/browser don't work out of the box, but I guess that's fine.